quinn-proto/src/transport_parameters.rs

```diff
@@ -39,16 +47,22 @@ macro_rules! apply_params {
 
 macro_rules! make_struct {
     {$($name:ident ($code:expr) = $default:expr,)*} => {
+        /// Transport parameters used to negotiate connection-level preferences between peers
         #[derive(Debug, Copy, Clone, Eq, PartialEq)]
         pub struct TransportParameters {
```

Comment:
Can we make the fields `pub(crate)`? I feel like they're details we don't need to leak.

Reply:
Isn't that the default anyway?

Reply:
Ah no, never mind this.